### PR TITLE
[IMP] mass_mailing: differentiate mailing name from emails subject

### PR DESCRIPTION
**Pull request by tde-banana-odoo**
Task: https://www.odoo.com/web#id=1917602&action=333&active_id=965&model=project.task&view_type=form&menu_id=4720

### CODE REVIEW

File: addons/mass_mailing/views/mass_mailing_views.xml [hunk 1]
```diff
@@ -591,10 +591,8 @@
                         </div>
 
                         <group>
-                            <label for="name" string="Subject"/>
-                            <div>
-                                <field name="name" nolabel="1"/>
-                            </div>
+                            <field name="name" required="1" string="Mailing Name"/>
```
Comment (jem-odoo) [inline addons/mass_mailing/views/mass_mailing_views.xml]: Isn't the name already required ? (as it came from utm.source)

File: addons/mass_mailing/models/mass_mailing.py [hunk 2]
```diff
@@ -451,6 +451,7 @@ def default_get(self, fields):
         return res
 
     active = fields.Boolean(default=True)
+    subject = fields.Char('Subject', help='Subject of emails to send', required=True)
```
Comment (jem-odoo) [inline addons/mass_mailing/models/mass_mailing.py]: Don't forget the migration tips, as we are adding a required field. I think populate it with the name is a good start ;)

File: addons/mass_mailing/views/mass_mailing_views.xml (hunk 1 above)
Comment (tde-banana-odoo) [reply addons/mass_mailing/views/mass_mailing_views.xml]: To my surprise it was not.

File: addons/mass_mailing/models/mass_mailing.py (hunk 2 above)
Comment (tde-banana-odoo) [reply addons/mass_mailing/models/mass_mailing.py]: Seems right. Copy name into subject for existing mailings seem correct.

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**